docs/embedded/patmos.mdx

```diff
@@ -7,101 +7,121 @@ Lingua Franca's C-runtime supports [Patmos](https://github.com/t-crest/patmos),
+LIB_PATH := ~/t-crest/local/patmos-unknown-unknown-elf/lib
+SERIAL?=/dev/ttyUSB0
+
+INCS := -I"$(LF_PROJECT_ROOT)/include" \
+	-I"$(LF_PROJECT_ROOT)/include/api" \
+	-I"$(LF_PROJECT_ROOT)/include/core" \
+	-I"$(LF_PROJECT_ROOT)/include/$(LF_MAIN_TARGET)" \
+	-I"$(CURDIR)/include/$(LF_MAIN_TARGET)" \
+	-I"$(CURDIR)/include/core/modal_models" \
+	-I"$(CURDIR)/include/core/utils" \
+	-I"$(CURDIR)/tag/api"\
+	-I"$(CURDIR)/low_level_platform/api" \
+	-I"$(CURDIR)/logging/api" \
+	-I"$(CURDIR)/version/api" \
+	-I"$(CURDIR)/platform/api" \
+	
+CC := patmos-clang 
+CFLAGS := -O2 $(INCS) -DINITIAL_EVENT_QUEUE_SIZE=10 -DINITIAL_REACT_QUEUE_SIZE=10 -DLF_SINGLE_THREADED=0 -DLF_REACTION_GRAPH_BREADTH=1
+SRC_FILES := _$(LF_MAIN_TARGET_LC)_main.c $(LF_MAIN_TARGET).c lib/schedule.c
+SRC_FILES += core/reactor_common.c core/lf_token.c core/reactor.c core/tag.c core/environment.c 
+SRC_FILES += core/utils/util.c core/utils/vector.c  
+SRC_FILES += core/utils/pqueue.c core/utils/pqueue_tag.c core/utils/pqueue_base.c
+SRC_FILES += core/utils/hashset/hashset_itr.c core/utils/hashset/hashset.c 
+SRC_FILES += core/clock.c low_level_platform/impl/src/lf_atomic_patmos.c 
+SRC_FILES += low_level_platform/impl/src/lf_patmos_support.c
+
+OBJ_FILES := $(patsubst %.c,%.o,$(SRC_FILES)) 
+EXE_NAME := $(LF_MAIN_TARGET).elf
+
+.PHONY: all clean wcet
+
+all: $(EXE_NAME)
+
+# Target for the executable
+$(EXE_NAME): $(OBJ_FILES)
+	$(CC) $(CFLAGS) -o $@ $^ -mserialize=$(PML_FILE_NAME).pml
+# Rule to compile C files into object files
+$(OBJ_FILES): %.o: %.c
+	$(CC) $(CFLAGS) -c -o $@ $<
+
+clean:
+	rm -f $(OBJ_FILES) $(EXE_NAME)
+
+wcet: $(EXE_NAME)
+	platin wcet --disable-ait -i $(PML_FILE_NAME).pml -b $(EXE_NAME) -e _helloworld_mainreaction_function_0 --report report.txt
+	
+```
 
-1. `pasim` a software ISA simulator that is written in C++.
-2. `patemu` a cycle-accurate hardware emulator generated from the hardware description.
+Then move this Makefile inside the `src-gen/HelloWorld` folder and run the following make command:
 
-To execute reactions on Patmos, the [Patmos toolchain](https://github.com/t-crest/patmos) needs
-to be installed. The web page contains a quick start, detailed information including how to
-perform WCET analysis is available in the
-[Patmos Reference Handbook](http://patmos.compute.dtu.dk/patmos_handbook.pdf).
+    make -C src-gen/HelloWorld
 
-To execute the "hello world" reactor on Patmos use the LF compiler to generate the C code.
-Compile the reactor with the Patmos compiler (in `src-gen`):
+Since LF still doesn't support Patmos officialy, you need to copy some files inluding `lf_patmos_support` c and h files in the related folders before executing make command whether manually or by executing the following shell file that automates the copying process (considering those files are located in a folder called `files`). If you choose shell file, you can also add Makefile to the list. 
```

Comment:
I don't believe this is still accurate?

Reply:
Yes, probably this is not anymore accurate.